dpypx/autodraw.py

```diff
@@ -21,20 +21,16 @@ class AutoDrawer:
     @classmethod
     def load_image(
             cls, client: Client, xy: tuple[int, int],
-            image: Image.Image, scale: float = 1) -> AutoDrawer:
-        """Draw from the pixels of an image."""
-        if image.mode == 'RGBA':
-            new_image = Image.new('RGB', image.size)
-            new_image.paste(image, mask=image)
-            image = new_image
+            image: Image.Image, scale: float = 1, strict: bool = False) -> AutoDrawer:
+        image = image.convert(mode="RGBA")
         width = round(image.width * scale)
         height = round(image.height * scale)
         # Since the image is only loaded once we can afford to use a high
         # quality filter.
         resized = image.resize((width, height), Image.LANCZOS)
         data = list(resized.getdata())
         grid = [
-            [Pixel(*pixel) for pixel in data[start:start + width]]
+            [(Pixel(*pixel[:3]), bool(round(pixel[3] / 255))) for pixel in data[start:start + width]]
```

Comment:
```suggestion
            [(Pixel(*pixel[:3]), pixel[3] > 127) for pixel in data[start:start + width]]
```
Equivalent code, less computation and simpler.

Reply:
Even so actually this line is longer than ideal. Perhaps
```suggestion
            [
                (Pixel(*pixel[:3]), bool(round(pixel[3] / 255)))
                for pixel in data[start:start + width]
            ]
```